staging/src/k8s.io/bgd-controller/bgd.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: demo.google.com/v1
+kind: BlueGreenDeployment
+metadata:
+  name: blue-green-deployment
+  namespace: default
+  labels:
+    app: nginx
+spec:
+  replicas: 1 # determine number of RS replicas
```

Comment:
Should change of `.spec.replicas` field of BGD object affects number of RS replicas?